docs/Contributing/architecture/software/vulnerability-management.md

```diff
@@ -0,0 +1,95 @@
+# Vulnerability Management Architecture
```

Comment:
This article in its current form is subtly inaccurate/incomplete. We have public and contributor vuln management articles and should link to those from the base readme instead of having this article at all.

There are potentially useful workflows etc. that should be documented in contributor docs for vulns, but this addresses none of those.

Reply:
Updating with links to the guides. Emptying except for headers.